index.html

```diff
@@ -6516,7 +6580,17 @@ <h3>Executing Script</h3>
    <dd><p>Return the <a data-lt="deserialize a shadow root">deserialized</a>
     <a>shadow root</a> of <var>value</var>.
 
-   <dt>instance of <a>Array</a>
+  <dt><a>Object</a> that <a>represents a web frame</a>
+  <dd>
+    <p>Return the <a data-lt="deserialize a web frame">deserialized</a>
+      <a>web frame</a> of <var>value</var>.
+
+  <dt><a>Object</a> that <a>represents a web window</a>
+  <dd>
+    <p>Return the <a data-lt="deserialize a web window">deserialized</a>
+      <a>web window</a> of <var>value</var>.
+
+    <dt>instance of <a>Array</a>
```

Comment:
nit: indentation seems off (compared to the existing indentation)

Reply:
Yeah. I accidentally saved with formatting and that broke it. Will fix it in my next push.